src/util/cpupinning.cpp

```diff
@@ -0,0 +1,80 @@
+#ifdef __LINUX__
+#include <errno.h>
+#include <pthread.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#include <QFile>
+#endif
+
+#include <QThread>
+#include <QtDebug>
+
+#include "util/cpupinning.h"
+
+namespace mixxx {
+
+#ifdef __LINUX__
+bool CpuPinning::moveThreadToCpuset(const QString& cgroup) {
+    qDebug() << "Move task" << gettid() << "to cpuset" << cgroup;
```

Comment:
This dos not build. 
/home/daniel/workspace/advanced_autodj_2/src/util/cpupinning.cpp:21:32: error: ‘gettid’ was not declared in this scope
     qDebug() << "Move task" << gettid() << "to cpuset" << cgroup;

Reply:
including this works: 
```
#if __GLIBC__ == 2 && __GLIBC_MINOR__ < 30
#include <sys/syscall.h>
#define gettid() syscall(SYS_gettid)
#endif
```
found here: 
https://stackoverflow.com/questions/21279649/getting-error-in-c-program-undefined-reference-to-gettid